docs/logs/getting-started-aspnetcore/Program.cs

```diff
@@ -6,25 +6,21 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
-// Remove default providers and add OpenTelemetry logging provider.
-// For instructional purposes only, disable the default .NET console logging provider to
-// use the verbose OpenTelemetry console exporter instead. For most development
-// and production scenarios the default console provider works well and there is no need to
+// For instructional purposes only, disable the default .NET logging providers.
+// We remove the console logging provider in this demo to use the verbose
+// OpenTelemetry console exporter instead. For most development and production
+// scenarios the default console provider works well and there is no need to
 // clear these providers.
 builder.Logging.ClearProviders();
 
-builder.Logging.AddOpenTelemetry(logging =>
-{
-    var resourceBuilder = ResourceBuilder
-        .CreateDefault()
-        .AddService(builder.Environment.ApplicationName);
-
-    logging.SetResourceBuilder(resourceBuilder)
-
-        // ConsoleExporter is used for demo purpose only.
-        // In production environment, ConsoleExporter should be replaced with other exporters (e.g. OTLP Exporter).
-        .AddConsoleExporter();
-});
+// Add OpenTelemetry logging provider by calling the WithLogging extension.
```

Comment:
should we add a note here to warn users that this approach only works if their exporter also supports this model?

Reply:
Might get confusing. I think either a) we leave it as is or b) we show the old style. Let me know which you prefer! I'm hoping the gap is short-lived and we can get these extensions added soon after 1.9.0 is out.

Reply:
okay to keep this as-is, assuming the gap is short-lived.